packages/json-schema/README.md

```diff
@@ -16,6 +16,14 @@ builder content schema:
 }
 ```
 
+## Setup
```

Comment:
```suggestion
## Local Setup
```
I think we might want to point out that these steps are not necessary in order to use the schema, but only if you want to generate it locally from.